components/Editor/WYSIWYGContainer.tsx

```diff
@@ -0,0 +1,50 @@
+import dynamic from 'next/dynamic';
+import 'react-quill/dist/quill.snow.css';
+
+const ReactQuill = dynamic(() => import('react-quill'), { ssr: false });
+
+interface WYSIWYGContainerProps {
+  content: string;
+  handleChange: (content: string, delta: any) => void;
+}
+
+const WYSIWYGContainer: React.FC<WYSIWYGContainerProps> = ({
+  content,
+  handleChange
+}) => {
+  const modules = {
+    toolbar: [
+      [{ size: ['small', false, 'large', 'huge'] }],
+      [{ font: [] }],
+      ['bold', 'italic', 'underline', 'strike'],
+      [
+        { align: [] },
+        { color: [] },
+        { background: [] },
+        'blockquote',
+        'code-block',
+        { list: 'ordered' },
+        { list: 'bullet' },
+        { script: 'sub' },
+        { script: 'super' }
+      ],
+      ['link', 'image'],
+      [{ indent: '-1' }, { indent: '+1' }],
+      ['clean']
+    ]
+  };
+
+  return (
+    <div className="h-screen w-full overflow-auto">
```

Comment:
i think it might be this overflow-auto that is causing the box to extend too much horizontally, correct me if im wrong tho :'0

Reply:
or maybe we can set the width of the editor bar to be fixed

Reply:
Fixed, good catch!